src/app.js

```diff
@@ -4,20 +4,32 @@
  */
 
 import { SharedMap } from "fluid-framework";
-import { AzureClient, LOCAL_MODE_TENANT_ID } from "@fluidframework/azure-client";
+import { AzureClient } from "@fluidframework/azure-client";
 import { InsecureTokenProvider } from "@fluidframework/test-client-utils"
 
-// The config is set to run against a local service by default. Run `npx tinylicious` to run locally
-// Update the corresponding properties below with your tenant specific information to run against your tenant.
+// The config is set to run against a local service by default.
 const serviceConfig = {
     connection: {
-        tenantId: LOCAL_MODE_TENANT_ID, // REPLACE WITH YOUR TENANT ID
-        tokenProvider: new InsecureTokenProvider("" /* REPLACE WITH YOUR PRIMARY KEY */, { id: "userId" }),
-        orderer: "http://localhost:7070", // REPLACE WITH YOUR ORDERER ENDPOINT
-        storage: "http://localhost:7070", // REPLACE WITH YOUR STORAGE ENDPOINT
+        type: "local",
+        tokenProvider: new InsecureTokenProvider("" , { id: "userId" }),
+        endpoint: "http://localhost:7070",
```

Comment:
is this supposed to be called serviceEndpoint?

Reply:
nope, we went with just endpoint. It's within the `connection` section of a `seviceConfig`. serviceEndpoint was a bit redundant.

Reply:
Here's a link to the strongly type object. https://github.com/microsoft/FluidFramework/blob/23a6d93331624a411e97a19dd41fafe9df299338/azure/packages/azure-client/src/interfaces.ts#L69